shop4All-backend/src/main/java/com/example/shop4All_backend/services/UserService.java

```diff
@@ -30,18 +32,21 @@ public class UserService {
 
     final String passwordRegex = "^(?=.*[A-Z])(?=.*[a-z])(?=.*\\d)(?=.*[!@#$%^&*])[A-Za-z\\d!@#$%^&*]{8,}$";
     final String emailRegex = "^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+$";
+    private static final Logger logger = LoggerFactory.getLogger(UserService.class);
```

Comment:
The logger declaration should stand at the [top](https://stackoverflow.com/questions/54876281/what-should-be-the-position-of-logger-in-the-class#:~:text=I%20have%20a%20preference%20for,classes%20will%20be%20more%20standard.) (line 28). Do this for all declarations.